### PR TITLE
Updated Chimp Example Explanation for #2033 

### PR DESCRIPTION
**Pull request by that1guy232**
#2033

### CODE REVIEW

File: docs/reST/tutorials/en/chimp-explanation.rst [hunk 1]
```diff
@@ -78,8 +78,8 @@ First, we import the standard "os" python module. This allow
 us to do things like create platform independent file paths.
 
 In the next line, we import the pygame package. In our case, we import
-pygame as ``pg``, so that all of the functionality of pygame is able to
-be referenced from the namespace ``pg``.
+pygame, all of the functionality of pygame is able to
+be referenced from this package.
```
Comment (Matiiss) [inline docs/reST/tutorials/en/chimp-explanation.rst:82]: I think it's better to just leave it at
"On the next line, we import the pygame package."

The other sentence made sense before, but now it feels out of place.

Comment (that1guy232) [reply docs/reST/tutorials/en/chimp-explanation.rst:82]: I agree, when I made the change i wanted to keep it as close to the original but reading it again, It does feel out of place.

Comment (that1guy232) [reply docs/reST/tutorials/en/chimp-explanation.rst]: Updated with the changes.

**Review by Matiiss — APPROVED** [review]
LGTM
Although not sure what to do with "In the next line" as it seems that "On the next line" would fit better, but I'll leave that up to you.

**Review by Starbuck5 — APPROVED** [review]
Thanks!

**Pull request merged**